<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1010</width>
    <height>698</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>1011</width>
      <height>681</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="image">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-10</y>
        <width>1011</width>
        <height>691</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="adminBtn">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>280</y>
        <width>181</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 26pt &quot;Poor Richard&quot;;
color: rgb(255, 255, 255);
background-color: rgb(0, 139, 136);</string>
      </property>
      <property name="text">
       <string>Admin</string>
      </property>
     </widget>
     <widget class="QPushButton" name="doctorBtn">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>380</y>
        <width>181</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 26pt &quot;Poor Richard&quot;;
color: rgb(255, 255, 255);
background-color: rgb(0, 139, 136);</string>
      </property>
      <property name="text">
       <string>Doctor</string>
      </property>
     </widget>
     <widget class="QPushButton" name="reciptionistBtn">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>480</y>
        <width>181</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 26pt &quot;Poor Richard&quot;;
color: rgb(255, 255, 255);
background-color: rgb(0, 139, 136);</string>
      </property>
      <property name="text">
       <string>Receptionist</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="image2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-10</y>
        <width>1011</width>
        <height>691</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>130</y>
        <width>441</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 36pt &quot;Algerian&quot;;
color: rgb(0, 0, 255);</string>
      </property>
      <property name="text">
       <string>Please Login</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="username">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>220</y>
        <width>361</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 600 16pt &quot;Segoe UI Variable&quot;;</string>
      </property>
      <property name="placeholderText">
       <string>User name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="password">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>310</y>
        <width>361</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 18pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::EchoMode::Password</enum>
      </property>
      <property name="placeholderText">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QPushButton" name="loginBtn">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>420</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 700 26pt &quot;Perpetua&quot;;
color: rgb(255, 255, 255);
background-color: rgb(0, 139, 136);</string>
      </property>
      <property name="text">
       <string>login</string>
      </property>
     </widget>
     <widget class="QPushButton" name="back1">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>410</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 700 26pt &quot;Perpetua&quot;;
color: rgb(255, 255, 255);
background-color: rgb(0, 139, 136);</string>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
